public_html/wp-content/plugins/wordcamp-organizer-survey/includes/debrief-survey/page.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * Adds a page which contains the debrief survey.
+ */
+
+namespace WordCamp\OrganizerSurvey\DebriefSurvey\Page;
+
+defined( 'WPINC' ) || die();
+
+add_filter( 'pre_trash_post', __NAMESPACE__ . '\prevent_deletion', 10, 2 );
+add_filter( 'pre_delete_post', __NAMESPACE__ . '\prevent_deletion', 10, 3 );
+
+/**
+ * Constants.
+ */
+const SURVEY_PAGE_ID = 'organizer_debrief_survey_page';
+
+
+/**
+ * Return the page ID.
+ *
+ * @return mixed
+ */
+function get_page_id() {
+	return get_option( SURVEY_PAGE_ID );
+}
+
+
+/**
+ * Prevent deletion of the Survey page, unless force_delete is true.
+ *
+ * @param bool|null $check Whether to go forward with trashing/deletion.
+ * @param WP_Post   $post  Post object.
+ * @param bool      $force_delete Whether to bypass the trash, set when deactivating the plugin to clean up.
+ */
+function prevent_deletion( $check, $post, $force_delete = false ) {
+	$survey_page = (int) get_option( SURVEY_PAGE_ID );
+
+	if ( $survey_page === $post->ID ) {
+		// Allow it, and delete the option if the page is force-deleted.
+		if ( $force_delete ) {
+			delete_option( SURVEY_PAGE_ID );
+			return $check;
+		}
+
+		return false;
+	}
+
+	return $check;
+}
+
+/**
+ * Get the content of the survey page.
+ *
+ * @return string
+ */
+function get_page_content() {
+	return <<<EOT
+	<!-- wp:paragraph -->
+	<p>If you recently organized a WordPress event, please take the survey below to share your feedback about your experience.</p>
+	<!-- /wp:paragraph -->
+	
+	<!-- wp:jetpack/contact-form {"subject":"New feedback received from your website","style":{"spacing":{"padding":{"top":"16px","right":"16px","bottom":"16px","left":"16px"}}}} -->
+	<div class="wp-block-jetpack-contact-form" style="padding-top:16px;padding-right:16px;padding-bottom:16px;padding-left:16px"><!-- wp:group {"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group"><!-- wp:jetpack/field-text {"label":"What event did you organize?","required":true,"requiredText":"(required)"} /--></div>
+	<!-- /wp:group -->
+	
+	<!-- wp:group {"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group"><!-- wp:heading -->
+	<h2 class="wp-block-heading"><strong><strong><strong>Part I. <strong>General Information</strong></strong></strong></strong></h2>
+	<!-- /wp:heading -->
+	
+	<!-- wp:group {"layout":{"type":"constrained"}} -->
+	<div class="wp-block-group"><!-- wp:jetpack/field-text {"label":"How many tickets were sold?","required":true,"requiredText":"(required)"} /--></div>
```

Comment:
This can also be pulled from existing data. There may be other questions that can also be removed for that reason.

Reply:
In this PR, I've pre-populated the value for the question "What event did you organize?". The data sources for the other questions are different and might need to look for proper functions to retrieve the existing data. I'll handle that in a separate PR.